raiden/transfer/mediated_transfer/mediation_fee.py

```diff
@@ -93,18 +93,24 @@ def linspace(start: TokenAmount, stop: TokenAmount, num: int) -> List[TokenAmoun
 
 
 def calculate_imbalance_fees(
-    channel_capacity: TokenAmount, max_imbalance_fee: FeeAmount
+    channel_capacity: TokenAmount, proportional_imbalance_fee: RelativeFeeAmount
 ) -> Optional[List[Tuple[TokenAmount, FeeAmount]]]:
     """ Calculates a quadratic rebalancing curve.
 
-    The penalty term takes the value `max_imbalance_fee` at the extrema.
+    The penalty term takes the following value at the extrema:
+        (channel_capacity / 2) * (proportional_imbalance_fee / int(1e6)
     """
-    if max_imbalance_fee == 0:
+    if proportional_imbalance_fee == 0:
         return None
```

Comment:
I don't know when this function is supposed to return `None`.  Here it seems possible to return `[(somethihng, 0), (something, 0), ...]`.  Maybe the docstring can explain when it's `None`?

Reply:
In that case it's true, we could just return two points as you write above. However in the case `channel_capacity == 0:` we don't support a return list with only one element, so we chose to return `None`  in this case.

Reply:
Yes, I just felt a bit random.  But I guess the callers are dealing with `None` reasonably.